package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "fxa-notification-server",
+  "version": "1.0.0",
+  "description": "This is an experimental hub service for handling account event notifications in the Firefox Accounts ecosystem.  Service Providers will be able to use it to publish notifications of various events, and Reliers will be able to use it to subscribe to said notifications.",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mozilla/fxa-notification-server.git"
+  },
+  "author": "",
+  "license": "MPL 2.0",
+  "bugs": {
+    "url": "https://github.com/mozilla/fxa-notification-server/issues"
+  },
+  "homepage": "https://github.com/mozilla/fxa-notification-server",
+  "dependencies": {
+    "convict": "0.6.1",
+    "hapi": "8.2.0",
+    "joi": "5.1.0",
+    "mozlog": "1.0.2"
+  },
```

Comment:
A bit late to the party here (let me know if you want a new bug), but it looks like there are marginally newer versions of **lab** (minor), **joi** (major), and **mozlog** (patch):

```sh
$ npm outdated --depth 0

Package  Current  Wanted  Latest  Location
lab        5.2.1   5.2.1   5.4.0  lab
joi        5.1.0   5.1.0   6.0.3  joi
mozlog     1.0.2   1.0.2   1.0.3  mozlog
```

I ran `$ npm shrinkwrap --dev` and then validated the shrinkwrap file via **nsp**, and it doesn't look like we have any sketchy/vuln modules, so probably not a big deal.